services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -178,6 +178,13 @@ async function putRecurringVivaCase(vivaPerson) {
   const casePersonCoApplicant = getUserByRole(casePersonList, 'coApplicant');
   if (casePersonCoApplicant) {
     caseItemPutParams.Item['GSI1'] = `USER#${casePersonCoApplicant.personalNumber}`;
+
+    const mainApplicantPno = stripNonNumericalCharacters(String(vivaPerson.case.client.pnumber));
+    appendEncryptionAttributes(
+      initialFormAttributes,
```

Comment:
Do not know where initialFormAttributes belongs from, but we should try to deal with this value in an immutable fashion. ie copy the current object and the return a new object.